xilem/src/view/label.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright 2024 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
-use masonry::{text::TextBrush, widget, ArcStr};
+use masonry::{text::ArcStr, text::TextBrush, widget};
```

Comment:
I would have expected this to merge in a `cargo fmt`: into `use masonry::{text::{ArcStr, TextBrush}, widget};`